### ISSUE TITLE
helm binary in cloud-native-basic (30970) has mismatched versions

### ISSUE THREAD

**Issue by eadamsintel** [clearlinux/distribution#1225]
I never could get influxdb installed through helm using Clear Linux's built in helm binary for the `cloud-native-basic bundle`.  I get network connection errors from inside the pod. However, if I install helm from `curl -L https://git.io/get_helm.sh | bash` and then initialize influxdb with the same yaml file it works.  I checked the version and noticed it was inconsistent between the client version and server version.  The one that installs from `https://git.io` has the same version for both client and server.

The script installs helm to `/usr/local/bin`

```
eadams@otc-tme-tp-01a~ $ /usr/bin/helm version
Client: &version.Version{SemVer:"v2.13+unreleased", GitCommit:"", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.14.3", GitCommit:"0e7f3b6637f7af8fcfddb3d2941fcc7cbebb0085", GitTreeState:"clean"}
eadams@otc-tme-tp-01a~ $ /usr/local/bin/helm version
Client: &version.Version{SemVer:"v2.14.3", GitCommit:"0e7f3b6637f7af8fcfddb3d2941fcc7cbebb0085", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.14.3", GitCommit:"0e7f3b6637f7af8fcfddb3d2941fcc7cbebb0085", GitTreeState:"clean"}
```

**Comment by mythi** [comment]
@jascott1?

**Comment by phmccarty** [comment]
The `helm` package in the distro is currently at version `2.13.1`, as you can see from the [package repo](https://github.com/clearlinux-pkgs/helm/blob/621def2ffe99ad4482a7d9f08861d146261b2b96/helm.spec).

Do we simply need to upgrade the `helm` package to latest to fix this issue?

**Comment by jascott1** [comment]
It is hard to say without knowing the issue. Helm deploys YAML so if OP is having inter pod network issues it would be because of misconfiguration. @eadamsintel we would need to use `helm template` for each version of Helm and compare if we wanted to understand why.

@phmccarty Upgrading the package is probably a good idea.

**Comment by eadamsintel** [comment]
This issue might just be that the version is not the latest and I am not using helm the correct way.  My assumption is that Clear Linux has or should have the latest version of each binary within reason.  I am trying some experiments with the built in clear Linux helm and the one from `git.io` to see if I can get the client/server version to match and see if that fixes my issues. When both are 2.14 then the pod will finish initializing but if the client is 2.13 and server is 2.14 then it does not.  It takes time to try things out between meetings. I might just be missing a step or using a wrong step when I install tiller and the versions are getting mismatched with the built in helm for Clear Linux.

**Comment by eadamsintel** [comment]
OK My eyes are failing me. Actually it put me back to where the client/server version still are mismatched. I saw a client version of `2.13.4` and a server of `2.14` so I just misread it. However, the influxdb did install this time which I can't explain. Maybe rebooting the server and re initializing everything cleared some broken ip table rule out or something or maybe I did things in a different order. Regardless, the client version in Clear Linux is still over a month behind the client version available upstream.
